exasol_script_languages_container_tool/lib/api/run_db_tests.py

```diff
@@ -13,11 +13,11 @@
 
 def run_db_test(flavor_path: Tuple[str, ...],
                 release_goal: Tuple[str, ...] = ('release',),
-                generic_language_test: Optional[Tuple[str, ...]] = None ,
-                test_folder: Optional[Tuple[str, ...]] = None,
-                test_file: Optional[Tuple[str, ...]] = None,
+                generic_language_test: Tuple[str, ...] = tuple(),
+                test_folder: Tuple[str, ...] = tuple(),
+                test_file: Tuple[str, ...] = tuple(),
                 test_language: Tuple[str, ...] = (None,),
```

Comment:
That is wierd?

Reply:
corresponding click default value is `[None]`:
see https://github.com/exasol/script-languages-container-tool/blob/1ee272b47194e6251de662036abfbe945bfe3835/exasol_script_languages_container_tool/cli/commands/run_db_tests.py#L37